fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/DataApiRequestImpl.java

```diff
@@ -372,13 +373,13 @@ public DataApiRequestImpl() {
         super();
         this.table = null;
         this.granularity = null;
-        this.dimensions = null;
+        this.dimensions = Collections.emptySet();
```

Comment:
For now, you can go back to setting granularity to a default, but Deprecate this method and make a note in the ChangeLog that we need to remove this method entirely, please.